ui/pages/confirmations/components/simulation-details/simulation-details.tsx

```diff
@@ -367,6 +400,16 @@ export const SimulationDetails: React.FC<SimulationDetailsProps> = ({
           balanceChanges={incoming}
           testId="simulation-rows-incoming"
         />
+        {alertModalVisible && (
+          <MultipleAlertModal
```

Comment:
Just confirming this is compatible with any other alerts so we can navigate between them still?